Empire/Empire.cpp

```diff
@@ -1458,6 +1498,8 @@ int Empire::StockpileID(ResourceType res) const {
         return m_capital_id;
         break;
     case RE_INDUSTRY:
+        return IMPERIAL_OBJECT_ID;
```

Comment:
Or should I rip out the current stockpile system as it is now (as it is not used). I could maybe also use the empire id as identifier (see the discussion with dbenage-cx above)